app/controllers/companies_controller.rb

```diff
@@ -39,12 +39,36 @@ def update
 
   def destroy
     company = Company.find(params[:id])
-    company.delete
+    company.destroy
 
     flash[:success] = "#{company.name} was successfully deleted!"
     redirect_to companies_path
   end
 
+  def dashboard
+    @top_three_companies = find_top_three_by_average
+    @jobs_by_level_of_interest = group_jobs_by_level_of_interest
+    @jobs_by_location = group_jobs_by_location
+    render :dashboard
+  end
+   
+  def find_top_three_by_average
```

Comment:
is there a more appropriate place for these methods? do they maybe belong in a model?!